clients/src/main/java/org/apache/kafka/clients/admin/internals/ListConsumerGroupOffsetsHandler.java

```diff
@@ -110,41 +112,97 @@ public String apiName() {
                         groupOffsetsListing.put(topicPartition, new OffsetAndMetadata(offset, leaderEpoch, metadata));
                     }
                 } else {
-                    log.warn("Skipping return offset for {} due to error {}.", topicPartition, error);
+                    handlePartitionError(groupId, topicPartition, error, groupsToUnmap, groupsToRetry);
```

Comment:
v0 and v1 do not have the top level error so it is indeed correct to check them here. Could you put a small comment to explain this here? Could you also mark KAFKA-13064 as a blocker for 3.0? This is a regression, I think.

Reply:
Marked KAFKA-13064 as blocker, and add comments:
`// In responseData V0-V7, there's no group level error, we have to handle partition errors here`
Thanks.

Reply:
Sorry, it should be v0-v1, I'll update it.